main.js

```diff
@@ -86,35 +81,38 @@ function initMainWindow() {
     // (this triggers smaller breakpoints) - this size should cause
     // the same layout results on all OSes
     // minWidth/minHeight: both need to be specified or none will work
-    width:     1270,
-    minWidth:  1270,
-    height:    675,
+    width: 1270,
+    minWidth: 1270,
+    height: 675,
     minHeight: 675,
-    icon:      path.join(__dirname, 'resources/icon.png'),
+    icon: path.join(__dirname, 'resources/icon.png'),
 
     webPreferences: {
       webviewTag: false,
       nodeIntegration: false,
       sandbox: true,
       contextIsolation: false,
-      preload: path.join(__dirname, 'preload.js')
-    },
+      preload: path.join(__dirname, 'preload.js'),
+      webSecurity: !options.dev
```

Comment:
This line is the only change in this file (the remainder are formatting changes).